README.md

```diff
@@ -0,0 +1,34 @@
+# PostgreSQL Snap
+[![Release to Snap Store](https://github.com/canonical/postgresql-snap/actions/workflows/release.yaml/badge.svg)](https://github.com/canonical/postgresql-snap/actions/workflows/release.yaml)
+
+This repository contains the packaging metadata for creating a snap of PostgreSQL built from the official Ubuntu repositories.  For more information on snaps, visit [snapcraft.io](https://snapcraft.io/). 
+
+## Installing the Snap
+The snap can be installed directly from the Stap Store.  Follow the link below for more information.
+<br>
+
+[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/postgresql)
+
+## Building the Snap
+### Clone Repository
+```bash
+git clone git@github.com:canonical/postgresql-snap.git
+cd postgresql-snap
+```
+### Installing and Configuring Prerequisites
+```bash
+sudo snap install snapcraft
+sudo snap install lxd
+sudo lxd init --auto
+```
+### Packing and Installing the Snap
+```bash
+snapcraft pack
+sudo snap install ./postgresql*.snap --devmode
+```
+
+## License
+The PostgreSQL Snap is free software, distributed under the Apache
+Software License, version 2.0. See
+[LICENSE](https://github.com/canonical/charme-postgresql-snap/blob/14.0/edge/licenses/LICENSE-snap)
```

Comment:
1) typo? should be charmed [in the first place](https://github.com/canonical/charmed-postgresql-snap/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R31)?
Please PR there.

2) should be without charmed here. :-)

Reply:
Fixed the link and started https://github.com/canonical/charmed-postgresql-snap/pull/85 in the charmed snap repo.